os/kernel/debug/dbg_internal.h

```diff
@@ -0,0 +1,67 @@
+/****************************************************************************
```

Comment:
File name is too common and it does not represent the purpose.
You already use memdbg.c so that memdbg.h is better than this I think.

Reply:
modified